doc/electrs-rest-btcd-1.0.0.yaml

```diff
@@ -0,0 +1,936 @@
+              schema:
+                type: array
+                items:
+                  type: string
+  /block/{hash}/header:
+    get:
+      tags:
+        - block
+      summary: Returns the Hex Encoded block header
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Block header
+          content:
+            text/plain:
+              schema:
+                type: string
+  /block/{hash}/raw:
+    get:
+      tags:
+        - block
+      summary: Returns the binary respresentation of the block
+      description: Responses can be cahced indefinitely.
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Raw block data
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+  /block/{hash}/hex:
+    get:
+      tags:
+        - block
+      summary: Get raw block data by hash in hex
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Raw block data
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+  /block/{hash}/txid/{index}:
+    get:
+      tags:
+        - block
+      summary: Returns up to 25 transactions based on an optional starting index.
+      description: Responses for this endpoint can be cached indefinitely. 
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: index
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Transaction ID
+          content:
+            text/plain:
+              schema:
+                type: string
+  /block/{hash}/txs:
+    get:
+      tags:
+        - block
+      summary: Get transactions in a block starting from an index
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: start_index
+          in: query
+          required: false
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Transactions for a given block
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Transaction'
+  /address-prefix/{address_prefix}:
+    get:
+      tags:
+        - address
+      summary: Get address stats by prefix
+      description: |
+        Elements-based chains don't have the `{funded,spent}_txo_sum` fields.
```

Comment:
Omit any elements information from this spec

Reply:
removed.